grails-app/views/_core_theme_includes.gsp

```diff
@@ -1,4 +1,6 @@
 <link rel="stylesheet" type="text/css" href="${resource(dir: 'js', file: 'ext-3.3.1/resources/css/ext-all-notheme.css')}"/>
 <link rel="stylesheet" type="text/css" href="${resource(dir: 'js', file: 'ext-3.3.1/resources/css/xBaseTheme.css')}"/>
 
-<link rel="shortcut icon" href="${resource(dir: 'images', file: 'favicon.ico')}" type="image/x-icon"/>
+<g:if test="${grailsApplication.config.portal.icon}" >
```

Comment:
The condition is useful here. If there is no `grailsApplication.config.portal.icon` there will be no broken tag created. The browser will still look in the root directory for a file called `favicon.ico` which could be desirable.